index.html

```diff
@@ -7,15 +7,15 @@
   <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet"/>
   <link href="https://fonts.googleapis.com/css?family=Roboto+Mono" rel="stylesheet"/>
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet"/>
-  <link href="/static/app.css" rel="stylesheet"/>
-  <link href="/static/favicon.ico" rel="icon" type="image/x-icon"/>
+  <link href="/app.css" rel="stylesheet"/>
+  <link href="/favicon.ico" rel="icon" type="image/x-icon"/>
 </head>
```

Comment:
Of course. Actually it would be good, if all static assets (that is everything but the index.html) were served from a certain subfolder. So I move everything (also the app.js and so on) into the `static/` folder